index.js

```diff
@@ -359,8 +360,14 @@ DumbPipe.registerOpener("camera", function(message, sender) {
     audio: false,
   }, function(localMediaStream) {
     mediaStream = localMediaStream;
+    url = URL.createObjectURL(localMediaStream);
 
-    video.src = URL.createObjectURL(localMediaStream);
+    video.onerror = video.onloadeddata = () => {
```

Comment:
Nit: we're trying to maintain as much compatibility with Chrome as possible, since their developer tools have some features that are missing in Firefox (like the ability to break on an unresponsive script), so we've been avoiding Firefox-specific JS features like fat-arrow functions.